From: Jason Rhinelander <jason@imaginary.ca>
Date: Mon, 16 Mar 2020 01:31:30 -0300
Subject: Devendor lokimq

---
 httpserver/CMakeLists.txt | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/httpserver/CMakeLists.txt b/httpserver/CMakeLists.txt
index f33420f..f3a4311 100644
--- a/httpserver/CMakeLists.txt
+++ b/httpserver/CMakeLists.txt
@@ -26,8 +26,6 @@ loki_add_subdirectory(../storage storage)
 loki_add_subdirectory(../utils utils)
 loki_add_subdirectory(../pow pow)
 loki_add_subdirectory(../crypto crypto)
-set(BUILD_SHARED_LIBS OFF CACHE BOOL "disable shared libraries") # Tells loki-mq to do a static build
-loki_add_subdirectory(../vendors/loki-mq loki-mq)
 find_package(OpenSSL REQUIRED)
 
 set(Boost_LIBRARIES "${CMAKE_SOURCE_DIR}/boost/lib/libboost_filesystem.a;${CMAKE_SOURCE_DIR}/boost/lib/libboost_system.a;${CMAKE_SOURCE_DIR}/boost/lib/libboost_thread.a;-lpthread;${CMAKE_SOURCE_DIR}/boost/lib/libboost_program_options.a;${CMAKE_SOURCE_DIR}/boost/lib/libboost_chrono.a;${CMAKE_SOURCE_DIR}/boost/lib/libboost_date_time.a;${CMAKE_SOURCE_DIR}/boost/lib/libboost_atomic.a")
@@ -39,7 +37,6 @@ target_link_libraries(httpserver_lib PUBLIC utils)
 target_link_libraries(httpserver_lib PUBLIC pow)
 target_link_libraries(httpserver_lib PUBLIC resolv)
 target_link_libraries(httpserver_lib PUBLIC crypto)
-target_link_libraries(httpserver_lib PUBLIC lokimq::lokimq)
 
 set_property(TARGET httpserver_lib PROPERTY CXX_STANDARD 14)
 
@@ -60,6 +57,11 @@ target_compile_definitions(httpserver PRIVATE -DSTORAGE_SERVER_GIT_HASH_STRING="
 target_compile_definitions(httpserver PRIVATE -DSTORAGE_SERVER_BUILD_TIME="${BUILD_TIME}")
 
 find_package(PkgConfig QUIET)
+
+pkg_check_modules(LOKIMQ REQUIRED liblokimq>=1.0.0)
+target_link_libraries(httpserver PUBLIC ${LOKIMQ_LIBRARIES})
+target_include_directories(httpserver PUBLIC ${LOKIMQ_INCLUDE_DIRS}/lokimq)
+
 if(PKG_CONFIG_FOUND)
     pkg_check_modules(SYSTEMD libsystemd)
     # Default ENABLE_SYSTEMD to true if we found it
